pkg/api/v1beta3/dynakube/activegate_props.go

```diff
@@ -0,0 +1,121 @@
+package dynakube
```

Comment:
Can we move the files to a `properties` directory and then have the names without `_props` suffix?

Reply:
different folder == different package

you can't define methods for `packageA.MyStruct` in `packageB`